DV-11/src/Transliteration.java

```diff
@@ -0,0 +1,282 @@
+    vocabulary.put("OJSC", "ОАО");
+    vocabulary.put("CJSC", "ЗАО");
+    vocabulary.put("LLC", "ООО");
+    vocabulary.put("Ltd.", "ООО");
+    vocabulary.put("SRL", "ООО");
+    vocabulary.put("fond", "фонд");
+    vocabulary.put("Fond", "Фонд");
+    vocabulary.put("SEE", "ГОУ");
+    vocabulary.put("MEE", "МОУ");
+    vocabulary.put("NEE", "НОУ");
+    vocabulary.put("union", "союз");
+    vocabulary.put("Union", "Союз");
+    vocabulary.put("eyeye", "еее");
+    vocabulary.put("shch", "щ");
+    vocabulary.put("Shch", "Щ");
+    vocabulary.put("yuye", "юе");
+    vocabulary.put("Yuye", "Юе");
+    vocabulary.put("yaye", "яе");
+    vocabulary.put("Yaye", "Яе");
+    vocabulary.put("aye", "ае");
+    vocabulary.put("Aye", "Ае");
+    vocabulary.put("eye", "ее");
+    vocabulary.put("Eye", "Ее");
+    vocabulary.put("iye", "ие");
+    vocabulary.put("Iye", "Ие");
+    vocabulary.put("oye", "ое");
+    vocabulary.put("Oye", "Ое");
+    vocabulary.put("uye", "уе");
+    vocabulary.put("Uye", "Уе");
+    vocabulary.put("yye", "ые");
+    vocabulary.put("ye", "ье");
+    vocabulary.put("ia", "ья");
+    vocabulary.put("iy", "ий");
+    vocabulary.put("Iy", "Ий");
+    vocabulary.put("zh", "ж");
+    vocabulary.put("Zh", "Ж");
+    vocabulary.put("kh", "х");
+    vocabulary.put("Kh", "Х");
+    vocabulary.put("ts", "ц");
+    vocabulary.put("Ts", "Ц");
+    vocabulary.put("ch", "ч");
+    vocabulary.put("Ch", "Ч");
+    vocabulary.put("sh", "ш");
+    vocabulary.put("Sh", "Ш");
+    vocabulary.put("yu", "ю");
+    vocabulary.put("Yu", "Ю");
+    vocabulary.put("ya", "я");
+    vocabulary.put("Ya", "Я");
+    return vocabulary;
+  }
+
+  /**
+   * Initialization vocabulary single symbols from cyrylic ti latin
+   * @param vocabulary hashmap where vocalulary is.
+   * @return vocabulary with symbols in haspmap form.
+   */
+  private HashMap<String, String> initializationRU_EN(HashMap<String, String> vocabulary) {
+    vocabulary.put("А", "A");
+    vocabulary.put("Б", "B");
+    vocabulary.put("В", "C");
+    vocabulary.put("Г", "G");
+    vocabulary.put("Д", "D");
+    vocabulary.put("Е", "E");
+    vocabulary.put("Ё", "E");
+    vocabulary.put("Ж", "Zh");
+    vocabulary.put("З", "Z");
+    vocabulary.put("И", "I");
+    vocabulary.put("Й", "Y");
+    vocabulary.put("К", "K");
+    vocabulary.put("Л", "L");
+    vocabulary.put("М", "M");
+    vocabulary.put("Н", "N");
+    vocabulary.put("О", "O");
+    vocabulary.put("П", "P");
+    vocabulary.put("Р", "R");
+    vocabulary.put("С", "S");
+    vocabulary.put("Т", "T");
+    vocabulary.put("У", "U");
+    vocabulary.put("Ф", "F");
+    vocabulary.put("Х", "Kh");
+    vocabulary.put("Ц", "Ts");
+    vocabulary.put("Ч", "Ch");
+    vocabulary.put("Ш", "Sh");
+    vocabulary.put("Щ", "Shch");
+    vocabulary.put("Ы", "Y");
+    vocabulary.put("Э", "E");
+    vocabulary.put("Ю", "Yu");
+    vocabulary.put("Я", "Ya");
+    vocabulary.put("а", "a");
```

Comment:
this is quite expensive to store Capital and non capital variations, please, find more elegant way to address capitalization